lcls-twincat-motion/Library/POUs/Motion/FB_MotionPneumaticActuator.TcPOU

```diff
@@ -0,0 +1,209 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.3">
+  <POU Name="FB_MotionPneumaticActuator" Id="{3965ca68-046c-4858-93f1-aaf5b6059f9e}" SpecialFunc="None">
+    <Declaration><![CDATA[(*This function blcok implements a pnuematic actuator. That can be signle or double acting by setting the ibSingleCntrl accordingly*)
+(* with double acting ibCntrlHold signal should be false, while with single acting the signal should be true*)
+FUNCTION_BLOCK FB_MotionPneumaticActuator
+VAR_INPUT
+    (*EPS Interlock Bits*)
+    ibInsertOK: BOOL; (*Actuator can be Inserted*)
+    ibRetractOK: BOOL; (*ACtuator can be retracted*)
+    ibPMPS_OK:BOOL; (*to be linked the Arbiter bit*)
+    ibSingleCntrl:BOOL;(* TRUE if Actuator requires one Output signal to be activated, FALSE if its double acting i.e two outputs are required*)
+    ibCntrlHold:BOOL; (* Control Signal must retain its value, must be TRUE in the case of single acting*)
+    ibOverrideInterlock:BOOL; (*if true interlocks are ignored*)
+    // Reset fault
+    {attribute 'pytmc' := '
+    pv: FF_Reset
+    '}
+    i_xReset: BOOL;
+END_VAR
+VAR_OUTPUT
+    {attribute 'pytmc' := '
+    pv:
+    '}
+    stPeumaticActuator    :    DUT_MotionPneumaticActuator;
```

Comment:
Typo here?

Reply:
argh, now I have to fix it everywhere.
Good catch. Thanks!